spec/models/like_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+
+feature 'Liking photos' do
+
+  context 'Signed in' do
+
+    let!(:sea) {Photo.create(title:'Awesome sea',
+                             image: File.new("spec/features/sea.jpg") )}
+
+    scenario "User can like a photo, which updates the photo 'like' count", js: true do
+
+      sign_up
+      click_link 'Like'
+      expect(page).to have_content '1 like'
+      expect(page).not_to have_content '0 likes'
+    end
+  end
+
+  context 'Not signed in' do
+
+    let!(:sea) {Photo.create(title:'Awesome sea',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Space missing after colon.